recipes/pyogrio/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "pyogrio" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # no official release yet, for now use archive fallback
+  url: https://github.com/brendan-ward/pyogrio/archive/3e5174ab8250c4b1b0c84e0c8fbb916f9f839564.zip
+  sha256: 025adc6703602826ec8529a3de581c9735b9539ddba5d9d39cf5042b8d33efa9
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<38 or win or python_impl == 'pypy']
```

Comment:
When I bumped my local docker image from 3.7 to 3.8 this built without a problem. Is the minimum really 3.8? Just curious on what features you are using that are not on 3.7.

Reply:
>  Is the minimum really 3.8? Just curious on what features you are using that are not on 3.7.

I suppose it's compatible with py 3.6/3.7 as well, but for now only tested on py 3.8. 
But, the original PR build it for all python versions, but was also failing for py3.7. So limiting to py3.8 today was more as an attempt to get it passing ;)

Reply:
Nah. That was a red-herring. The problem was the numpy version in staged-recipes.